rust/unicode/src/tables.rs

```diff
@@ -1498,3 +1498,6 @@ pub const LINEBREAK_STATE_MACHINE: [u8; 3827] = [
     16, 17, 146, 147, 148, 149, 150, 151, 152, 153, 26, 27, 28, 29, 30, 159,
     160, 161, 162, 163, 36, 165, 166, 167, 168, 169, 170,
 ];
+
```

Comment:
we'll want to add `#[cfg_attr(rustfmt, rustfmt_skip)]` to these tables, so they're ignored when #846 lands.

Reply:
OK! I have added.